docs/UserGuide.md

```diff
@@ -70,6 +70,9 @@ Adds an animal to the address book.
 Format: `add n/NAME i/ID g/SEX s/SPECIES b/BREED db/DOB da/DOA`
 
 * ID must be a unique 4 digit number.
+* SEX must be either `male` or `female`.
```

Comment:
I think it would be good to specify whether case-sensitive or not here!

Reply:
Yep. That sounds better!